tests/aws/scenario/mythical_mysfits/test_mythical_misfits.snapshot.json

```diff
@@ -0,0 +1,683 @@
+{
+  "tests/aws/scenario/mythical_mysfits/test_mythical_misfits.py::TestMythicalMisfitsScenario::test_deployed_infra_state": {
+    "recorded-date": "12-10-2023, 16:56:10",
+    "recorded-content": {
+      "outputs": {
+        "ClicksBucketDestinationName": "<clicks-bucket-destination-name:1>",
+        "DeliveryStreamArn": "arn:aws:firehose:<region>:111111111111:deliverystream/<delivery-stream-name:1>",
+        "DeliveryStreamName": "<delivery-stream-name:1>",
+        "<physical-resource-id:1>Name": "<physical-resource-id:1>",
+        "PopulateDbFunctionName": "<populate-db-function-name:1>",
+        "StreamProcessorFunctionName": "<stream-processor-function-name:1>",
+        "UserClicksServiceAPIEndpoint": "https://<user-clicks-service-a-p-i-id:1>.execute-api.<region>.amazonaws.com/<physical-resource-id:7>/",
```

Comment:
nit: once we run the test against LS, we might need to re-work the pattern here

Reply:
yes, transformer would be nice. but we can iterate over this as well. There might be other cases we haven't considered yet, we will see once we make it work with LS

>  I think CFn might return the right endpoint for LocalStack 

i think so too :)